### PR TITLE
Updating "Doxygen Header" to match example packages.

### PR DESCRIPTION
**Pull request by remod**
Closing https://git.anybotics.com/anybotics/anybotics/-/issues/4753

### CODE REVIEW

**Comment by hogabrie** [comment]
LGTM

File: cppguide.html [hunk 1]
```diff
@@ -4609,8 +4609,8 @@ <h4 class="stylepoint_subsection">Doxygen Header</h4>
 Every file must contain a doxygen header of the following form. In .cpp files the brief section can be omitted.
 <pre>
 /**
- * @authors     Max Muster
- * @affiliation ANYmal Research Institution
+ * @file        MyClass.hpp
```
Comment (roversch) [inline cppguide.html]: Just `@file` without postfix will also work by the way. It picks up the filename automagically.

Comment (remod) [reply cppguide.html]:
```suggestion
 * @file
```

Like this? That would certainly prevent copy/paste mistakes in the future :)

Comment (roversch) [reply cppguide.html]: exactly. See [doxygen documentation](http://epic-alfa.kavli.tudelft.nl/share/doc/doxygen-1.6.1/html/commands.html#cmdfile).

**Pull request merged**